lib/kafka/avro_producer.rb

```diff
@@ -42,7 +42,8 @@ def encode_payload(schema, payload)
       magic_byte = [0].pack('C')
       # NOTE: This is a placeholder schema ID. In a real-world scenario, this should be fetched from a schema registry
       # ID = 5 is the Event Bus schema ID for test schema, replace this with the actual schema ID when running locally
-      schema_id_bytes = [5].pack('N') # should be schema id
+      schema_id_bytes = [1].pack('N') # should be schema id
```

Comment:
Could you remove this from the PR? Unfortunately needs to remain hard-coded as 5 for the time being.